homeassistant/components/device_automation/__init__.py

```diff
@@ -25,11 +27,23 @@
 )
 
 TYPES = {
-    "trigger": ("device_trigger", "async_get_triggers"),
-    "condition": ("device_condition", "async_get_conditions"),
-    "action": ("device_action", "async_get_actions"),
+    "trigger": (
+        "device_trigger",
+        "async_get_triggers",
+        "async_get_trigger_capabilities",
+    ),
+    "condition": (
+        "device_condition",
+        "async_get_conditions",
+        "async_get_condition_capabilities",
+    ),
+    "action": ("device_action", "async_get_actions", "async_get_action_capabilities"),
 }
 
+POSITIVE_TIME_DELTA_DICT = vol.All(
```

Comment:
Why is this a constant? Why shouldn't we use the config_validation helper directly ?

Reply:
It's not a dict btw :P

Reply:
It's a constant because this is the specific combination we have taught `custom_serializer` about and which is supported by the frontend.

Reply:
Properly fixed now, the issue was that schemas were validated twice.